components/ai_chat/resources/page/state/data-context-provider.tsx

```diff
@@ -206,6 +252,31 @@ function DataContextProvider (props: DataContextProviderProps) {
     getPageHandlerInstance().pageHandler.managePremium()
   }
 
+  const setInputText = (text: string) => {
+    setInputText_(text)
+
+    if (text.startsWith('/')) {
+      setIsToolsMenuOpen(true)
+
+      // effectively remove the leading slash before comparing it to the action labels.
+      const searchText = text.substring(1).toLocaleLowerCase()
+
+      const filteredList = ACTIONS_LIST.map(category => ({
+        ...category,
+        actions: category.actions.filter(action => {
+          // actionType with -1 is a subcategory so we skip
+          if (action.type === -1) return
```

Comment:
If you end up with a separate heading you could check `if (!('type' in action)) return` or `if (action.type === 'heading') return` or whatever

Reply:
updated